packages/marko-web-search/loaders/search.js

```diff
@@ -66,12 +66,15 @@ module.exports = async ({ apolloBaseCMS, apolloBaseBrowse } = {}, {
   opSuffix,
 } = {}) => {
   if (!apolloBaseCMS || !apolloBaseBrowse) throw new Error('Both the BaseCMS and Base Browse Apollo clients must be provided.');
+  const query = (!searchQuery.match(/^".+"$/) && !searchQuery.match(/^'.+'$/)) ? searchQuery : '';
+  const phrase = (searchQuery.match(/^".+"$/) || searchQuery.match(/^'.+'$/)) ? searchQuery : '';
```

Comment:
@Shinsina I believe you will also want to strip the wrapping quotes before setting searchQuery here. 
something like 
```markoJs
const phrase = (searchQuery.match(/^".+"$/) || searchQuery.match(/^'.+'$/)) ? searchQuery.replace(/^["'](.+(?=["']$))["']$/, '$1') : '';
```